apps/accounts/const/automation.py

```diff
@@ -49,9 +49,9 @@ class SecretStrategy(models.TextChoices):
 
 
 class SSHKeyStrategy(models.TextChoices):
-    add = 'add', _('Append SSH KEY')
-    set = 'set', _('Empty and append SSH KEY')
     set_jms = 'set_jms', _('Replace (Replace only keys pushed by JumpServer) ')
+    set = 'set', _('Empty and append SSH KEY')
+    add = 'add', _('Append SSH KEY')
 
 
 class TriggerChoice(models.TextChoices, TreeChoices):
```

Comment:
I'm sorry, but your message is not clear on what you're asking about specific code differences for a different topic. If this concerns programming related to secret or SSH key strategies with Django models in Python, please share which parts of the existing version you need checked and updated according to your needs. I can provide some guidance on Python coding best practices if needed!